dashboard/config/locales/scripts.en.yml

```diff
@@ -347,6 +347,13 @@ en:
           description_short: ''
           description: ''
           Internet Simulator: Internet Simulator
+          Unit 1: Unit 1
+          Unit 2: Unit 2
+          Sending Bits: Sending Bits
+          Routing and DNS: Routing and DNS
+          'Unit 1: Sending Bits': 'Unit 1: Sending Bits'
+          'Unit 2: Routing and DNS': 'Unit 2: Routing and DNS'
+          Internet Simulator Templates: Internet Simulator Templates
```

Comment:
Some of these were auto-generated as I worked, and no longer used.  Do I need to manually remove them?